sky/cli.py

```diff
@@ -3476,6 +3485,15 @@ def storage_delete(names: List[str], all: bool):  # pylint: disable=redefined-bu
         names = [s['name'] for s in storages]
     else:
         names = _get_glob_storages(names)
+    if not yes:
+        storage_names = ', '.join(names)
+        storage_str = 'storages' if len(names) > 1 else 'storage'
+        click.confirm(
+            f'Deleting {len(names)} {storage_str}: '
+            f'{storage_names}. Proceed?',
+            default=True,
+            abort=True,
+            show_default=True)
```

Comment:
When I run `sky storage delete -a`, a redundant `Deleting all storage objects.` is printed.
```
(base) ➜  sky-experiments git:(issue2708) ✗ sky storage delete -a             
Deleting all storage objects.
Deleting 1 storage: romil-test-bucket. Proceed? [Y/n]: ^CAborted!
```

Can we instead just show:
```
(base) ➜  sky-experiments git:(issue2708) ✗ sky storage delete -a             
Deleting <count> storages: romil-test-bucket, ... . Proceed? [Y/n]:
```

Reply:
Same when a storage name is specified:
```
(base) ➜  ~ sky storage delete romil-test-bucket
Deleting 1 storage object.
Deleting 1 storage: romil-test-bucket. Proceed? [Y/n]:
```

Reply:
This has been fixed, and `_get_glob_storages` has been modified accordingly.